templates/layouts/_base.html

```diff
@@ -149,7 +147,8 @@
         "secondsTextSingular": _("second"),
         "secondsTextPlural": _("seconds"),
         "endWithFullStop": true
-      })
-    }}
-  {% endif %}
+    })
+}}
+{# djlint:on #}
+{% endif %}
```

Comment:
Think this should be indented inside the block

Reply:
This becomes flattened every time we run djlint, so an issue with the library not indenting it correctly.

Reply:
See if moving the `{# djlint:on #}`/`{# djlint:off #}` outside of the if makes a difference

Reply:
That worked, might have to add some more excluded blocks to prevent some bad indents after format.